alerter/alerter.java

```diff
@@ -15,13 +19,17 @@ static void alertInCelcius(float farenheit) {
             // let us keep a count of failures to report
             // However, this code doesn't count failures!
             // Add a test below to catch this bug. Alter the stub above, if needed.
-            alertFailureCount += 0;
+            alertFailureCount += 1;

```

Comment:
This is the bug-fix. But the test still fails, because the test is wrong. 500 is given in both calls, so the failure count will be 2.
This assignment is about making the test strong enough to catch the bug. Put back the bug, and change the test code to exercise both the 200 and 500 paths